python/cudf/cudf/utils/utils.py

```diff
@@ -9,9 +9,9 @@
 
 import rmm
 
-mask_dtype = np.dtype(np.int8)
+mask_dtype = np.dtype(np.int32)
```

Comment:
Strictly speaking, we use `uint32`.

Reply:
Yea I tried, but there's situations we feed this into normal column ops where we don't yet support `uint` types and it caused issues.

Reply:
This scares me. Masks are just bits -- unsigned.